src/styles/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+
+  height: 134px;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  margin-top: 40px;
+
+  color: #060b35;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  margin-top: 4px;
+  display: block;
+
+  color: #616070;
+}
+
+.card__reviews-block {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+  align-items: center;
+}
+
+.card__rating {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 24px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__buy-button {
+  background: #00acdc;
+  border-radius: 5px;
+  text-align: center;
+}
+
+.card__buy-link {
+  display: block;
+  text-decoration: none;
+  padding: 12px;
+  margin-top: 16px;
+
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  color: #fff;
+}
+
+.card:hover .card__buy-link {
```

Comment:
Do you really need `.card__buy-link` here?

Reply:
I've applied new styles to the link, so I think yes, I need it